src/software/pipeline/main_cameraInit.cpp

```diff
@@ -166,7 +166,7 @@ int aliceVision_main(int argc, char** argv)
 
     // user optional parameters
     std::string defaultCameraModelName;
-    std::string allowedCameraModelsStr = "pinhole,radial1,radial3,brown,fisheye4,fisheye1";
+    std::string defaultDistortionModelName;
```

Comment:
these 2 should have a default value
like:
```
std::string defaultCameraModelName = camera::EINTRINSIC_enumToString(camera::EINTRINSIC::PINHOLE);
```

Reply:
it is unused in fact. But it's outside the scope of this PR.